{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2024-24255"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "A Race Condition discovered in geofence.cpp and mission_feasibility_checker.cpp in PX4 Autopilot 1.14 and earlier allows attackers to send drones on unintended missions."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-362"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://github.com/Drone-Lab/PX4-Autopilot/blob/report-the-faliure-of-precheck/report-the-faliure-of-precheck.md",
					"tags": [
						"Exploit",
						"Third Party Advisory"
					],
					"url": "https://github.com/Drone-Lab/PX4-Autopilot/blob/report-the-faliure-of-precheck/report-the-faliure-of-precheck.md"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "HIGH",
				"attackVector": "NETWORK",
				"availabilityImpact": "LOW",
				"baseScore": 4.2,
				"baseSeverity": "MEDIUM",
				"confidentialityImpact": "NONE",
				"integrityImpact": "LOW",
				"privilegesRequired": "LOW",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:L/A:L",
				"version": "3.1"
			},
			"exploitabilityScore": 1.6,
			"impactScore": 2.5
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:dronecode:px4_drone_autopilot:*:*:*:*:*:*:*:*",
						"versionEndIncluding": "1.14.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-02-15T18:46Z",
	"publishedDate": "2024-02-06T23:15Z"
}
